Seasonal/NewYear/YearBall.cs

```diff
@@ -0,0 +1,75 @@
+//cs_include Scripts/CoreBots.cs
+using Skua.Core.Interfaces;
+
+public class YearBall
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.SetOptions();
+
+        GetDrops();
+
+        Core.SetOptions(false);
+    }
+
+    public void GetDrops()
+    {
+        string[] ACNonmem = {
+            "AntiRetrograde Clock",
+            "Aurora Caller",
+            "Aurora Caller's Bow",
+            "Aurora Caller's Cape",
+            "Aurora Caller's Hat",
+            "Aurora Caller's Helm",
+            "Aurora Caller's Rune",
+            "Aurora Caller's Staff",
+            "Aurora Caller's Wings",
+            "Axeros' Snow Globe Wand",
+            "Chaotic Chrono Eye",
+            "Elegant Bejeweled Cane",
+            "Frigid Wolf Spear",
+            "Glacial Knight",
+            "Glacial Knight Cape",
+            "Glacial Knight Crown",
+            "Glacial Knight Crown Locks",
+            "Glacial Knight Hair",
+            "Glacial Knight Locks",
+            "Glacial Knight Rune",
+            "Glacial Knight Rune Sword Cape",
+            "Glacial Knight Shag",
+            "Glacial Knight Sword",
+            "Glacial Knight Swords",
+            "Nascent ChronoWeaver's Runes",
+            "New Year Dawning",
+            "Party Time Twig Pet",
+            "Sheathed Glacial Knight Cape",
+            "Timeseeker's Blades",
+            "Timeseeker's Staff"
+        };
+
+        if (!Core.isSeasonalMapActive("newyear"))
+            return;
```

Comment:
Preferably place this above the initiation of the array that this is currently under.
No need to save something to memory if the map ain't even active and its gonna stop anyway

Reply:
done 👍